frontend/scripts/docker-entrypoint.sh

```diff
@@ -19,8 +19,9 @@ declare -a ENV_VARIABLES=(
   "VUE_APP_PENDO_KEY"
   "VUE_APP_HOTJAR_KEY"
   "VUE_APP_ENV"
-  "VUE_APP_PIZZLY_URL"
-  "VUE_APP_PIZZLY_PUBLISHABLE_KEY"
+  "VUE_APP_NANGO_URL"
+  "VUE_APP_TYPEFORM_ID"
+  "VUE_APP_TYPEFORM_TITLE"
```

Comment:
I believe we are no longer using typeform, this was removed in this [PR](https://github.com/CrowdDotDev/crowd.dev/pull/619)
Not sure if this was some merge mistake. If so, could you remove where it was added again?

Reply:
Thanks for noticing this! I should simply remove them, right?